tests/ui/unused_unit.rs

```diff
@@ -143,4 +143,10 @@ mod issue14577 {
             todo!()
         }
     }
-}
\ No newline at end of file
+}
+
+mod pr14962 {
+    #[allow(unused_parens)]
```

Comment:
Since this is a targeted lint, could you please use `#[expect(unused_parens)]` instead? That doubles as an extra free test for `unused_parens` that would force us to think about it if its behavior changes.